pkg/api/realtime_purge/realtime_purge.go

```diff
@@ -0,0 +1,50 @@
+package realtime_purge
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/aziontech/azion-cli/pkg/cmd/version"
+	sdk "github.com/aziontech/azionapi-go-sdk/realtimepurge"
+)
+
+type Client struct {
+	apiClient *sdk.APIClient
+}
+
+func NewClient(c *http.Client, url string, token string) *Client {
+	conf := sdk.NewConfiguration()
+	conf.HTTPClient = c
+	conf.AddDefaultHeader("Authorization", "token "+token)
+	conf.AddDefaultHeader("Accept", "application/json;version=3")
+	conf.UserAgent = "Azion_CLI/" + version.BinVersion
+	conf.Servers = sdk.ServerConfigurations{
+		{URL: url},
+	}
+
+	return &Client{
+		apiClient: sdk.NewAPIClient(conf),
+	}
+}
+
+func (c *Client) Purge(ctx context.Context, urlToPurge []string) error {
+	var purg sdk.PurgeUrlRequest
+	purg.SetUrls(urlToPurge)
+	purg.SetMethod("delete")
+	request := c.apiClient.RealTimePurgeApi.PurgeUrl(ctx).PurgeUrlRequest(purg)
+
+	httpRes, err := c.apiClient.RealTimePurgeApi.PurgeUrlExecute(request)
+	if err != nil {
+		responseBody, _ := ioutil.ReadAll(httpRes.Body)
+		return fmt.Errorf("%w: %s", err, responseBody)
+	}
+
+	if httpRes.StatusCode != 201 {
+		return fmt.Errorf("%w: %s", err, httpRes.Status)
+	}
+
+	fmt.Println("\nDomain cache was purged")
```

Comment:
I'd suggest moving this Println to the Publish method itself.
There we can use whatever output was provided by the factory to print:
Something like:
```
		err = cmd.purgeDomains(f, domainReturnedName)
		if err != nil {
			return err
		}
                fmt.Fprintln(cmd.f.IOStreams.Out, "\nDomain cache was purged") // Added 
```

Reply:
done